General/TweakingBigImages.md

```diff
@@ -0,0 +1,116 @@
+# Tweaking big images
+
+Pharo is special in the sense that the language and IDE are mixed. This mean that your application runs into the IDE.
+In some cases we might end-up Pharo images having a large memory consumption (aka Big Images). 
+The goal of this page is to sum up some tweaking that can be done on Pharo images to improve the performances on such images. 
+
+* [Fast dragging](#fast-dragging)
+* [Disable taskbar previews](#disable-taskbar-previews)
+* [Tune Garbage Collection](#tune-garbage-collection)
+* [Disable Quality Assistant plugin](#disable-quality-assistant-plugin)
+
+## Fast dragging
+
+While dragging windows in Pharo an animation happens. This animation is not performant and can make the dragging of windows slow in large images.
+
+It is possible to workaround this problem by enabling a "fast dragging" option:
+
+```Smalltalk
+UITheme currentSettings fastDragging: true
+```
+
+This option is reset when we change the theme Pharo and should thus be executed each time you update the theme.
+
+## Disable taskbar previews
+
+While going oven the taskbar of Pharo, previews of the images are displayed. This feature can be really slow on big images. 
+It is possible to disable it:
+
+```Smalltalk
+TaskbarMorph showWindowPreview: false
+```
+
+## Tune Garbage Collection 
+
+Pharo is a language with a memory management using garbage collection. 
+When objects are created, memory is allocated to them. Once in a while, the memory is cleaned and the "garbage collector" free the memory of objects that are not been used anymore.
+
+This process can be customized via multiple parameters. To understand the impact of those parameters we explain here how the garbage collection works.
+
+Pharo memory (aka heap) is divided into multiple spaces:
```

Comment:
```suggestion
Because most objects usage stops quickly after their instantiation (we say they have a short life-time), Pharo memory (aka "the heap") is divided into multiple spaces:
```